wp-stripe-plaid.php

```diff
@@ -91,7 +91,7 @@ function deactivate_wp_stripe_plaid() {
  */
 require WP_STRIPE_PLAID_PATH . 'includes/class-wp-stripe-plaid.php';
 require WP_STRIPE_PLAID_PATH . 'admin/class-wp-stripe-plaid-settings-api.php';
-require WP_STRIPE_PLAID_PATH . 'vendor/autoload.php';
```

Comment:
Little confused here. We still need the Stripe SDK to call the Stripe API which is autoloader form the composer vendor directory.

Reply:
@noahott

Reply:
I had some trouble activating the plugin initially, and this seemed to solve it for me. Testing it again now I can no longer reproduce the error, so this change probably isn't needed.